File: Python/Warmup-1/makes10.py

# this function return true if one of the two given int values is ten or their sum is ten
def makes10(a, b):
	bool = False

	if a == 10 or b == 10 or a + b == 10:
		bool = True

	return bool

print(makes10(9, 10))
print(makes10(9 ,9))
print(makes10(1, 9))

File: Java/Warmup-1/makes10.java

public class warmup1 {

	// this function return true if one of the two given int values is ten or their sum is ten
	public boolean makes10(int a, int b) {
		
		boolean bool = false;
		
		if (a == 10 || b == 10 || a + b == 10)
			bool = true;
			
		return bool;

	}
	
	public static void main(String[] args) {
		
		warmup1 o = new warmup1();
		
		System.out.println(o.makes10(9, 10));
		System.out.println(o.makes10(9, 9));
		System.out.println(o.makes10(1, 9));

	}
	
}
